.gitlab-ci.yml

```diff
@@ -841,45 +840,51 @@ test_backend_integration:
       };
       trap handle_exit EXIT
 
-    - INTEGRATION_TEST_SUITE=$(integration/extra/release_tool.py --select-test-suite || echo "all")
-    - echo Running backend-tests suite $INTEGRATION_TEST_SUITE
-    - cd integration/backend-tests/
+    - INTEGRATION_TEST_SUITE=$(integration/extra/release_tool.py --select-test-suite)
 
-    # From 2.4.x on, the script would download the requirements by default
-    - if ./run --help | grep -e --no-download; then
-        RUN_ARGS="--no-download";
-      fi
+    - >
```

Comment:
Thanks for introducing this syntax in this piepeline file :clap: Hopefully it will inspire others to adopt it and do cleaner scripts.